katcp/test/test_resource_client.py

```diff
@@ -16,7 +24,7 @@
 import tornado
 import mock
 
-from thread import get_ident as get_thread_ident
+from _thread import get_ident as get_thread_ident
```

Comment:
I dont' know, that is what the Futurize tool did, and it seems to work. So I assume that is the correct way?